spec/system/nsm/additional_fees_spec.rb

```diff
@@ -229,6 +226,21 @@
         click_button 'Save changes'
         expect(page).not_to have_content('Adjustments')
       end
+
+      it 'can remove youth court fee adjustment through the adjustments tab' do
+        visit adjusted_nsm_claim_additional_fees_path(claim)
+
+        click_on 'Additional fees'
+
+        # might need to have a more specific selector
```

Comment:
How do we do this for letters and calls?

Something like (napkin Ruby)

```ruby
within(:xpath, "//tr[.//th[contains(text(),'Youth Court Fee')]]") do
  click_button("Delete")
end
```

Should work I think if there's no pre-existing standard

Reply:
Elsewhere in these system specs (e.g. disbursements_spec.rb) we use the pattern `within('.govuk-table__row', text: 'Youth court fee') do` which is marginally easier to read

Reply:
It just isn't tested in the letters_and_calls integration test (only the remove all uplifts button is)

Reply:
will try this though!

Reply:
That's fine then, Patrick's version is much cleaner

Reply:
ok - was having a lot of fun with xpath helper chrome extensions aha